docs/references/asset-canister.mdx

```diff
@@ -1,19 +1,20 @@
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-# Asset canister
+# Asset canisters
 
 <MarkdownChipRow labels={["Reference"]} />
 
 ## Overview
 
-The [asset canister](https://github.com/dfinity/sdk/tree/master/src/canisters/frontend/ic-frontend-canister) provides
-you with a mechanism to store and retrieve static assets from a canister deployed on the Internet
-Computer. It is generally used to serve HTML, JS, and CSS assets
-for [single page applications](https://en.wikipedia.org/wiki/Single-page_application), i.e. your application's frontend.
+An [asset canister](https://github.com/dfinity/sdk/tree/master/src/canisters/frontend/ic-frontend-canister), also referred to as a 'frontend canister' is a type of canister deployed on ICP that can be used to store and retrieve an application's frontend, typically used to serve static HTML, JS, and CSS assets.
 
-`dfx` provides this canister by default when your canister `type` is set to `assets` in `dfx.json`.
+To serve web content, a canister must implement a method that consumes an HTTP request (url, http method, and headers) and outputs an HTTP response (status, headers, and body). The canister method could return assets in the form of HTML, CSS, and Javascript content as part of the HTTP response.
 
-For example:
+A canister configured as `"type": "asset"` already contains the boilerplate code to facilitate these HTTP requests, making serving static content simple. Once the asset canister is deployed on ICP, the frontend interface can be accessed at http://<canister id>.ic0.app and http://<canister id>.raw.ic0.app.
```

Comment:
we've moved away from `ic0.app` and are encouraged to use `icp0.io` domains instead. this is to reduce reliance on Google, which owns the `.app` tld